platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -498,6 +500,40 @@ class MeasurementService {
     this._broadcastChange(this.EVENTS.MEASUREMENTS_CLEARED);
   }
 
+  jumpToMeasurement(viewportIndex, id) {
+    const measurement = this.measurements[id];
+
+    if (!measurement) {
+      log.warn(`No id provided, or unable to find measurement by id.`);
+      return;
+    }
+
+    this._addJumpToMeasurement(viewportIndex, id);
+
+    const eventName = this.EVENTS.JUMP_TO_MEASUREMENT;
+
+    const hasListeners = Object.keys(this.listeners).length > 0;
+    const hasCallbacks = Array.isArray(this.listeners[eventName]);
+
+    if (hasListeners && hasCallbacks) {
+      this.listeners[eventName].forEach(listener => {
+        listener.callback({ viewportIndex, measurement });
+      });
+    }
+  }
```

Comment:
Can't we re-use the pub/sub "broadcast" here? (instead of iterating listeners)

Reply:
The pub/sub broadcast broadcasts the same thing for every other event. I'd be happy to factor it out into a different function, but it can't be the same one.